bundle/manifests/falcon-helm-controller-manager-metrics-service_v1_service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  creationTimestamp: null
+  labels:
+    control-plane: controller-manager
+  name: falcon-helm-controller-manager-metrics-service
+spec:
+  ports:
+  - name: https
+    port: 8443
+    targetPort: https
```

Comment:
Where do we document that we require the installation of a web server?

Reply:
We don't require the installation of a web server. The operator manager pod provides this by default as part of how an operator provides metrics of the operator back to the cluster. Once again, this code is auto-generated.